src/main/java/com/videoplaza/dataflow/pubsub/TimeUtils.java

```diff
@@ -0,0 +1,18 @@
+package com.videoplaza.dataflow.pubsub;
+
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+
+public class TimeUtils {
```

Comment:
Add a private constructor to hide the implicit public one. (From sonar)

Reply:
Yea....About that.... :) Don't see the value in this. No one in their right mind would instantiate a TimeUtils class to call a static method and if it does for any yet unknown reason nothing bad will happen. There are cases when removing default constructor is important (e.g. when there is a static factory method, which is the only way to create an instance) but not these 2 guys here. However I can see how having an agreement to have those private ctors anyway can make things smoother :)